scripts/import/import_clinvar_xml

```diff
@@ -203,7 +203,10 @@ sub process_clinvar_set {
     warn "processing $record{Acc}\n" if $DEBUG == 1;
 
     ## recover from partial loads with file of accessions already loaded
-    next if $done{$record{Acc}};
```

Comment:
Just a small comment not related to the PR: this data comes from the file $done_file. This option should be included in L1333 to make clear which options are available.